src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -78,9 +78,18 @@ object ProtocPlugin extends AutoPlugin {
 
       val protocCache = TaskKey[FileCache[ModuleID]]("protoc-cache", "Cache of protoc executables")
 
+      val cacheArtifactResolver = SettingKey[Boolean](
+        "cache-artifact-resolver",
```

Comment:
I followed the pattern of `cache-classloaders`, but I wondered if this should be prefixed with `protoc-`